app/pages/collections/list.cjsx

```diff
@@ -24,33 +26,72 @@ List = React.createClass
           '/simple-avatar.jpg'
 
   cardLink: (collection) ->
-    [collection_owner, collection_name] = collection.slug.split('/')
-    "/collections/#{collection_owner}/#{collection_name}"
+    baseLink = "/"
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}/"
+    "#{baseLink}collections/#{collection.slug}"
 
   listCollections: ->
     query = {}
     if @props.params?.collection_owner?
       query.owner = @props.params.collection_owner
       query.include = 'owner'
-
+    if @props.project?
+      query.project_ids = @props.project.id
     query.favorite = @props.favorite
     Object.assign query, @props.location.query
 
     apiClient.type('collections').get query
 
   render: ->
     {location} = @props
-    <section className="resources-container collections-container">
+    classes = classNames {
+      "resources-container": true
+      "collections-container": true
+      "in-project-context": @props.project?
+    }
+
+    baseLink = if @props.project? then "/projects/#{@props.project.slug}/" else "/"
+    baseType = if @props.favorite then "favorites" else "collections"
+    shouldShowMoreLink = @props.params.profile_name? || @props.params.collection_owner?
+    showMoreLink = "#{baseLink}#{baseType}/"
```

Comment:
Re:
> Actually, it really makes the most sense to move the /users/name/collections link into the nav bar.

This takes us back to the "To the Zooniverse!" idea that was rejected last time. I'll put it back (I actually prefer it up there) but think we are going round in circles on this one!!